examples/scripts/custom_model.py

```diff
@@ -59,6 +59,12 @@
 ] = pybamm.electrolyte_conductivity.LeadingOrder(model.param)
 model.submodels["negative sei"] = pybamm.sei.NoSEI(model.param, "Negative")
 model.submodels["positive sei"] = pybamm.sei.NoSEI(model.param, "Positive")
+model.submodels[
+    "negative lithium plating"
+] = pybamm.li_plating.NoPlating(model.param, "Negative")
```

Comment:
This should be `pybamm.lithium_plating` (same in the line below). I think it is what is breaking the example tests.